docker/databricks/dbr/dbr9.1/genomics-with-hail/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# ===== For the runtime environment for this image we need the databricks azure setup ==============
+
+FROM projectglow/genomics:9.1 AS builder
+
+# ===== Set up Hail ================================================================================
+
+# earliest Hail version supported by Spark3 is 0.2.67
+ENV HAIL_VERSION=0.2.78
```

Comment:
match to HAIL_VERSION in build script

Reply:
Shouldn't this be 0.2.78 as this was tested on DBR 9.1 LTS?

Reply:
[build.sh](https://github.com/projectglow/glow/pull/513/files/9a095b32e7efc747ccf3e81345c37be0edcc44da#diff-5e88d12e5231077937292f2bf9dd8b156fd542f46ecb353fe52ced01931ff1f9) has the hail version tagged as `0.2.85`, please change that script to `0.2.78` to match this Dockerfile